backend/common/tests.py

```diff
@@ -0,0 +1,9 @@
+from common.utils.tests import TestCaseUtils
+
+
+class TestIndexView(TestCaseUtils):
+    view_name = "common:index"
+
+    def test_returns_status_200(self):
+        response = self.auth_client.get(self.reverse(self.view_name))
+        self.assertResponse200(response)
```

Comment:
I added this test just to check the CI was working correctly (both testing and coverage calculation)